api/v1beta1/agentserviceconfig_types.go

```diff
@@ -32,8 +32,6 @@ type OSImage struct {
 	Version string `json:"version"`
 	// Url specifies the path to the Operating System image.
 	Url string `json:"url"`
-	// rootFSUrl specifies the path to the root filesystem.
-	RootFSUrl string `json:"rootFSUrl"`
```

Comment:
I'm not sure we can remove this.

This is a breaking change and will prevent our operator from upgrading properly.

Reply:
ok, we can keep it for now then. do you know if we could remove it later without breaking upgradability?

Reply:
Not without an api version change I think.
cc @avishayt @mhrivnak

Reply:
Where is os-images used?  It's not in an API from what I can see.  Is it stored in the DB and that's the issue?

Reply:
It's a part of the agentserviceconfig CR.

Reply:
Here https://github.com/openshift/assisted-service/blob/86037f622038d984fffa517dc2b4f4df97667f7a/api/v1beta1/agentserviceconfig_types.go#L85

Reply:
I think we could still remove it from the swagger and DB, and mark it in the agentserviceconfig as ignored, to be removed in the next API version.

Reply:
Sure, marked RootFSUrl as deprecated in agentserviceconfig_types then.